src/chrome/content/rules/StartCom.xml

```diff
@@ -62,6 +56,6 @@ Fetch error: http://auth.startssl.com/ => https://auth.startssl.com/: (35, 'erro
 
   <!-- host startcom.org responds neither on 80 nor on 443,
     but should be protected from MitM and redirected to https://www.startcom.org -->
-  <rule from="^http://startcom\.org/" to="https://www.startcom.org/" />
```

Comment:
Do you know if there is a way to prevent the checker from failing at https://auth.startssl.com, which requires a client certificate?